sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/APISpec.groovy

```diff
@@ -61,6 +60,9 @@ class APISpec extends Specification {
     @Shared
     BlobContainerClient cc
 
+    @Shared
+    BlobContainerClient ccPremium
```

Comment:
Don't need this globally shared property, can scope it to the one test

Reply:
Did not see the comments here before merge. We can move it with some other changes. Sorry to miss your comment.